viewer/vueapp/src/components/connections/Connections.vue

```diff
@@ -647,6 +716,36 @@ export default {
         }
       });
     },
+    changeBaselineDate: function () {
+      this.$router.push({
+        query: {
+          ...this.$route.query,
+          baselineDate: this.query.baselineDate
+        }
+      });
+    },
+    changeBaselineVis: function () {
```

Comment:
do the same thing for baseline node visibility, but also update node, label, and link visibility based on which visibility option the user selects